Language/Generics/Superbounded_types/well_bounded_A05_t06.dart

```diff
@@ -6,7 +6,7 @@
 /// compile-time error.
 ///
 /// @description Checks that it is a compile-time error to declare a variable
-/// with a type which ia a non-function type alias with is not a well-bounded
+/// with a type which is a non-function type alias which is not well-bounded.
```

Comment:
Line 22 is somewhat surprising: No error, messages, and&mdash;true!&mdash;that type is indeed well-bounded (it's super-bounded). But what is it then doing in this test? ;-)

Reply:
Moved to `well_bounded_A05_t05.dart`